<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logfile</class>
 <widget class="QWidget" name="Logfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logfile</string>
  </property>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>500</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_remove">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>500</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_log">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>30</y>
     <width>644</width>
     <height>461</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>7</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>5</y>
     <width>577</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>plainTextEdit_log</tabstop>
  <tabstop>pushButton_remove</tabstop>
  <tabstop>pushButton_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
